nowcasting_dataset/geospatial.py

```diff
@@ -120,3 +122,46 @@ def calculate_azimuth_and_elevation_angle(
 
     # extract the information we want
     return solpos[["elevation", "azimuth"]]
+
+
+def get_osgb_center_from_osgb(
+    x_osgb: Union[xr.DataArray, List[float], np.ndarray],
+    y_osgb: Union[xr.DataArray, List[float], np.ndarray],
+) -> (float, float):
+    """
+    Get the center OSGB coords from a list of OSGB coords.
```

Comment:
Please add a line to the docstring to say that this function returns the `mean` of x_osgb and the `mean` of y_osgb (when I first read the function name, I assumed it selected the central location from the list of input locations... and my assumption was wrong :slightly_smiling_face: )

Reply:
thanks - ill update it